zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4347,37 +4308,45 @@ components:
       type: object
       properties:
         sort:
+          description: Sort field criteria.
           type: array
           items:
             allOf:
               - $ref: "#/components/schemas/SearchQuerySortRequest"
         page:
+          description: Pagination criteria.
           allOf:
             - $ref: "#/components/schemas/SearchQueryPageRequest"
           type: object
     SearchQueryPageRequest:
       type: object
       properties:
         from:
+          description: The index to start searching from.
           type: integer
           format: int32
         limit:
+          description: The maximum number of items to return in one request.
           type: integer
           format: int32
         searchAfter:
+          description: Items to search after. Correlates to the `lastSortValues` property of a previous search response.
           type: array
           items:
             type: object
         searchBefore:
+          description: Items to search before. Correlates to the `firstSortValues` property of a previous search response.
```

Comment:
Let me know if I got these descriptions incorrect. I took somewhat-educated guesses.

Reply:
As far as I can see, this is absolutely correct 👍